### PR TITLE
Dao

### PR DESCRIPTION
**Pull request by STOzaki**
Created Daos

### CODE REVIEW

**Review by al-amine — APPROVED** [review]
approved

File: src/com/lms/model/Branch.java [hunk 1]
```diff
@@ -33,35 +35,19 @@ public int getId() {
...
-		int result = 1;
-		result = prime * result + ((address == null) ? 0 : address.hashCode());
-		result = prime * result + id;
-		result = prime * result + ((name == null) ? 0 : name.hashCode());
-		return result;
+		return id;
 	}
 
 	@Override
-	public boolean equals(Object obj) {
```
Comment (al-amine) [inline src/com/lms/model/Branch.java]: it would be better if you override equals() using only the Id of the object without the other fields !

**Review by kingjon3377 — APPROVED** [review]
I generally approve; there are some mostly-cosmetic changes that I will make in my fork (removing redundant modifiers, making documentation use Javadoc syntax, etc.), but none of that should hold this back.

**Review by al-amine — APPROVED** [review]

File: src/com/lms/model/Branch.java (hunk 1 above)
Comment (STOzaki) [reply src/com/lms/model/Branch.java]: Yes, which Jonathan corrected. I updated this branch with those changes, so we should be good now.

**Comment by STOzaki** [comment]
> I generally approve; there are some mostly-cosmetic changes that I will make in my fork (removing redundant modifiers, making documentation use Javadoc syntax, etc.), but none of that should hold this back.

Well, can you make those changes and merge them in, like what you did before?

**Comment by kingjon3377** [comment]
> Well, can you make those changes and merge them in, like what you did before?

I can, but unlike my earlier PR to your fork (the changes I'm expecting to make to this code won't affect anything that uses it.

**Comment by STOzaki** [comment]
> > Well, can you make those changes and merge them in, like what you did before?
> 
> I can, but unlike my earlier PR to your fork (the changes I'm expecting to make to this code won't affect anything that uses it.

I guess that is up to you. It would be great if you can because you are amazing at commenting.

File: .classpath [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="src" path="src"/>
+	<classpathentry kind="src" path="lib"/>
+	<classpathentry kind="src" path="test"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.junit.JUNIT_CONTAINER/5"/>
+	<classpathentry kind="lib" path="C:/Users/salem/eclipse-workspace/NovaTechLMSSQL/lib/mysql-connector-java-5.1.47.jar"/>
```
Comment (kingjon3377) [inline .classpath:8]: This definitely needs to be changed from an absolute path to one relative to the project root.

Comment (STOzaki) [reply .classpath:8]: So, you mean something like lib/mysql-connector-java-5.1.47.jar?

Comment (kingjon3377) [reply .classpath:8]: Right. You used relative paths for the other libraries you added to `.classpath`, but this one won't work on anyone else's computer as it stands.

Comment (STOzaki) [reply .classpath:8]: You are right. I changed it and pushed it.

**Review by henryjyc — APPROVED** [review]

**Pull request merged**